db/data_migration/20170217153845_republish_publications_migration_two.rb

```diff
@@ -0,0 +1,18 @@
+document_scope = Document.where(
+  id: Publication.where(state: %w(draft published withdrawn)).pluck(:document_id)
+)
+
+lowest_id_for_this_republish = 260000
+highest_id_for_this_republish = 330000
+document_scope = document_scope.where(
```

Comment:
It was to avoid snarky remarks about 'magic numbers' from some of our more hilarious colleagues 😆